docs/FHIRServerUsersGuide.md

```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
-
-Note that the compressed file for the FHIR server installation ships with a version of the IBM&reg; Java 8 SDK that is already configured to support encryption.
-
-### 4.5.2 Use case
-A REST API client encrypts the body of the HTTP request (perhaps via a JAX-RS WriterInterceptor implementation), and sets the following http headers in the request:
-*	Content-Encoding – this request header should be set to the value `aescbc256` to indicate that the body is encrypted with the AES algorithm (in CBC mode) using a 256-bit encryption key.
-*	AES-Salt – this request header should be set to the 16-byte initialization vector (IV) used to encrypt the body. This byte sequence should be specified in 2-digit hex format.
-
-    Example: `000102030405060708090a0b0c0d0e0f`
-
-    Note: this value can be obtained by calling the `Cipher.getIV()` method after initializing the Cipher object with the encryption key.
-*	Accept-Encoding – this request header should be set to `aescbc256` if the API consumer wants the response body to be encrypted (using the same encryption key). The server will set the AES-Salt response header to the 16-byte IV used to encrypt the response body, similar to the way in which the API consumer sets it in the request.
-
-### 4.5.3 Request processing
-The encryption key is generated by the deployer of the FHIR server, and is shared with the API consumer (that is, both parties must use the same 256-bit AES encryption key; see [Section 4.5.5 Configuration](#455-configuration) for details).
-
-For incoming requests, if the servlet filter detects that the request body is encrypted (by examining the `Content-Encoding` and `AES-Salt` header values) it will decrypt the request body using the `AES/CBC/PKCS5Padding` algorthm (IBMJCEFIPS provider) along with the configured 256-bit encryption key and the IV found in the `AES-Salt` request header of the HTTP request, and then will provide the decrypted data to downstream filters and servlets (for example, the JAX-RS servlet) via the input stream.
-
-### 4.5.4 Response processing
-On the response path, if the `Accept-Encoding` header is set in the request and contains the string `aescbc256`, the servlet filter encrypts the response body using the same encryption algorithm, configured encryption key and a randomly-generated IV. The IV will be specified in the `AES-Salt` response header.
-
-The encrypted response data is then transmitted by the servlet container back to the API consumer, which decrypts the response body (perhaps via a JAX-RS ReaderInterceptor implementation), using the shared encryption key and the IV obtained from the `AES-Salt` header found in the HTTP response.
-
-### 4.5.5 Configuration
-The configuration of the encryption feature consists of the generation of an encryption key, plus the definition of a few properties in the `fhir-server-config.json` file.
```

Comment:
we should find and remove the associated properties in 
1. the java properties-handling constants/code
2. any fhir-server-config samples that have it
3. section 5 of this document (if they are there)

Reply:
updated 1/2/3